client/my-sites/plugins/plugins-navigation-header/index.jsx

```diff
@@ -192,7 +186,7 @@ const PluginsNavigationHeader = ( { navigationHeaderRef, categoryName, category,
 				<UploadPluginButton
 					isMobile={ isMobile }
 					siteSlug={ selectedSite?.slug }
-					hasUploadPlugins={ hasUploadPlugins }
+					hasUploadPlugins={ selectedSite }
```

Comment:
Nit: What do you think about making more explicitly you just want a boolean flag?

```suggestion
					hasUploadPlugins={ Boolean( selectedSite ) }
```

or 


```suggestion
					hasUploadPlugins={ !!selectedSite  }
```

Reply:
Thanks for highlighting this @WBerredo, I've actually been hesitating on this for a while, so it'd be best to discuss!  I started out with `!!selectedSite`, then saw that the selector's type is `SiteDetails | null | undefined`, so it could be used as is. Happy to cast it to make it more explicit though! 🙌